index.html

```diff
@@ -2713,7 +2713,913 @@ <h5>Question</h5>
+                        </li>
+                        <li>
+                            The code is public, but the list of contributors is small. No vulnerability reporting mechanism has been announced, or it's been announced but has no demonstrable track record.
+                        </li>
+                        <li>
+                            The code is partly private.
+                        </li>
+                        <li>
+                            The code is entirely private.
+                        </li>
+                    </ol>
+                </section>
+                <section>
+                    <h5>Relevance</h5>
+                    <p>
+                        Security vulnerabilities tend to be found and fixed best in code that has many active contributions and a strong history of correctly handled responsible disclosure.
+                    </p>
+                </section>
+                <section>
+                    <h5>Examples</h5>
+                    <table class="simple">
+                        <tr>
+                            <th>
+                                Method
+                            </th>
+                            <th>
+                                Net.
+                            </th>
+                            <th>
+                                Reg.
+                            </th>
+                            <th>
+                                Notes
+                            </th>
+                        </tr>
+                        <tr>
+                            <td>
+                                did:foo
+                            </td>
+                            <td>
+                                A
+                            </td>
+                            <td>
+                                D
+                            </td>
+                            <td>
+                                By design, did:foo supports 123.5 bits of security.
+                            </td>
+                        </tr>
+                    </table>
+                </section>
+            </section>
+            <section>
+                <h4>Diffuse Control</h4>
+                <section>
+                    <h5>Question</h5>
+                    <p>
+                        To what extent does the system support mechanisms where DID control is distributed across multiple parties (m-of-n control, threshold signatures, etc)?
+                    </p>
+                </section>
+                <section>
+                    <h5>Responses</h5>
+                    <ol type="A">
+                        <li>
+                            Rich diffuse control mechanisms are a first-class feature, and anyone who resolves a DID can see whether they are in use so confidence is increased. This may include device revocation (where keys must be exercised on a particular device to be valid).
```

Comment:
```suggestion
Rich diffuse control mechanisms are a first-class feature, and anyone who
resolves a DID can see whether they are in use so confidence is increased.
This may include device revocation (where keys must be exercised on a
particular device to be valid).
```